cpp/unreal_plugins/SimulationController/Source/SimulationController/StandaloneComponent.h

```diff
@@ -4,16 +4,17 @@
 
 #pragma once
 
-#include <type_traits> // is_base_of
+#include <concepts> // std::derived_from
 
+#include <Components/ActorComponent.h>
 #include <Components/SceneComponent.h>
 #include <Engine/World.h>
 #include <GameFramework/Actor.h>
 
-#include "CoreUtils/Assert.h"
-#include "CoreUtils/Unreal.h"
+#include "SpCore/Assert.h"
+#include "SpCore/Unreal.h"
```

Comment:
I think it's good to add `// CActorComponent` alongside this include since it's not straightforward as to where `CActorComponent` is defined.

Reply:
Great catch. 100% agreed.

Reply:
Fixed. There were some other instances where this style of header comment was missing for our own headers, so I fixed those too.